openrouteservice/src/main/java/heigit/ors/exceptions/MissingConfigParameterException.java

```diff
@@ -0,0 +1,8 @@
+package heigit.ors.exceptions;
+
+public class MissingConfigParameterException {
+
+    public MissingConfigParameterException(String missingConfigParameter) {
+        System.err.print("Config parameter " + missingConfigParameter + " is missing." + "\n");
```

Comment:
Would be better if this used the Log4j logging instead of standard System printing. (See ors/config/AppConfig.java for an example)

Reply:
I moved the error message handling to Log4j logging using LOGGING.info except one LOGGING.error for a malformed mail